pages/common/tbl.md

```diff
@@ -0,0 +1,24 @@
+# tbl
+
+> Format tables for troff.
+> More information: <https://manned.org/tbl>.
+
+- Format tables in troff:
```

Comment:
Why `in troff` and not `for troff`?
```suggestion
- Format tables for `troff` and print it to stdout:
```

Reply:
I meant it as kind of a "format the tables contained in the troff source".

Reply:
Oh, so it modifies the file?